ansible/configs/sap-smart/default_vars.yml

```diff
@@ -0,0 +1,531 @@
+## Common repositories
+rhel8_repos:
+   - "rhel-8-for-x86_64-baseos-rpms"
+   - "rhel-8-for-x86_64-appstream-rpms"
+   - "ansible-2-for-rhel-8-x86_64-rpms"
+
+rhel7_repos:
+  - "rhel-7-server-rpms"
+  - "rhel-7-server-satellite-6.7-rpms"
+  - "rhel-7-server-satellite-maintenance-6-rpms"
+  - "rhel-server-rhscl-7-rpms"
+  - "rhel-7-server-ansible-2.8-rpms"
+
+### Common Host settings
+
+install_ipa_client: false
+
+install_student_user: false
+
+# Do you want to run a full yum update
+update_packages: false
+
+## guid is the deployment unique identifier, it will be appended to all tags,
+## files and anything that identifies this environment from another "just like it"
+guid: defaultguid
+
+# This var is used to identify stack (cloudformation, azure resourcegroup, ...)
+project_tag: "{{ env_type }}-{{ guid }}"
+
+# This is where the ssh_config file will be created, this file is used to
+# define the communication method to all the hosts in the deployment
+deploy_local_ssh_config_location: "{{output_dir}}/"
+
+install_bastion: true
+install_common: true
+
+# Variable used to identify the request being made by a Red Hatter or Partner (external)
+rh_internal: true
```

Comment:
Since it seems based on the task that you are using this var for that you are concerned about the SAP software being made available to externals, would it not be safer to default this to `false` and then explicitly allow it in the agnosticv config for the internal catalog item - assuming you are planning on different ones?

Reply:
Yeah, agree